site/app/templates/forum/ForumBar.twig

```diff
@@ -87,29 +87,45 @@
+					</li>
 	          	{% endif %}
 	        {% endfor %}
 	        {% if core.getUser().accessGrading() %}
-              <li class="dropdown-divider"></li>
-              <li title="Click to edit categories" class='dropdown-item' role="menuitem">
-            	<a href ="{{ manage_categories_url }}" title="Edit Forum Categories" target="_blank" >Edit Categories</a>
+                <li class="divider"></li>
+              	<li title="Click to edit categories" class='dropdown-item' role="menuitem">
+            		<a href="{{ manage_categories_url }}" title="Edit Forum Categories" >Edit Categories</a>
+				</li>
 			{% endif %}
 			{% if thread_exists and not is_full_threads_page %}
-	          <li class="dropdown-divider"></li>
-	          <li title="Click to toggle all post attachments" class='dropdown-item' role="menuitem">
-	          	<a href="#" class="key_to_click" tabindex="0" onclick="loadAllInlineImages()">
-   					Attachments <span class="attachment-badge badge">{{ total_attachments }}</span>
-   				</a></li>
-	          <li class="dropdown-divider"></li>
-	              <li title="Sort posts by reply hierarchy" id="tree" class='dropdown-item' role="menuitem"><a class="key_to_click"  onclick="changeDisplayOptions('tree', {{ current_thread }})" href="#">Hierarchical</a></li>
-	              <li title="Sort posts by ascending chronological order" id="time" class='dropdown-item' role="menuitem"><a href="#" class="key_to_click" tabindex="0" onclick="changeDisplayOptions('time', {{ current_thread }})">Chronological  <i class="fas fa-angle-up"></i> </a></li>
-				        <li title="Sort posts by descending chronological order" id="reverse-time" class='dropdown-item' role="menuitem"><a href="#" class="key_to_click" tabindex="0" onclick="changeDisplayOptions('reverse-time', {{ current_thread }})">Chronological  <i class="fas fa-angle-down"></i> </a></li>
-	              {% if user_group <= core.getUser().accessGrading() %}
-	              	<li title="Sort posts by author's last name" id="alpha" class='dropdown-item' role="menuitem"><a href="#" class="key_to_click" tabindex="0" onclick="changeDisplayOptions('alpha')">Alphabetical</a></li>
-	              	<li title="Sort posts by author's registration section then last name" id="alpha_by_registration" class='dropdown-item' role="menuitem"><a href="#" class="key_to_click" tabindex="0" onclick="changeDisplayOptions('alpha_by_registration')">Alpha by Registration</a></li>
-	              	<li title="Sort posts by author's rotating section then last name" id="alpha_by_rotating" class='dropdown-item' role="menuitem"><a href="#" class="key_to_click" tabindex="0" onclick="changeDisplayOptions('alpha_by_rotating')">Alpha by Rotating</a></li>
-	              {% endif %}
+	          	<li class="divider"></li>
+	          	<li title="Click to toggle all post attachments" class='dropdown-item' role="menuitem">
+	          		<a href="#" class="key_to_click" tabindex="0" onclick="loadAllInlineImages()">
```

Comment:
This doesn't actually toggle all attachments. It will open all of them but clicking on this again does not close all attachments. This would probably be a separate issue and PR though.

Reply:
Good catch.  I agree that it should be another issue+PR.  I suspect that there might be other issues with this menu as well given how complex some of the code is.

Reply:
Opened #6475